packages/terra-functional-testing/src/services/wdio-visual-regression-service/methods/BaseCompare.js

```diff
@@ -105,7 +105,7 @@ class BaseCompare {
 
     // Added to allow for test reusability from terra repositories
     if (specPath.includes('node_modules')) {
-      [, specPath] = specPath.split('node_modules');
+      specPath = specPath.replace('node_modules', 'tests/wdio/__snapshots__');
```

Comment:
Should we use `path.join` to handle platform specific separator?
E.g. : `path.join('test, 'wdio', '__snapshots__')`

Reply:
updated here https://github.com/cerner/terra-toolkit/pull/644/commits/1f7eecf6329136adf78620af17774bb1d91ebf1c